packages/core/src/factory.ts

```diff
@@ -73,7 +74,13 @@ export function factory<Resource, Input, Output, Config>(
                     components.mocker.mock(
                       {
                         resource,
-                        input: { validations: { input: inputValidations }, data: input },
+                        input: {
+                          validations: {
+                            input: inputValidations,
+                            security: validateSecurity<Resource, Input>(input, resource),
```

Comment:
```suggestion
                            security: validateSecurity(input, resource),
```

TS should infer the types, right?

Reply:
Hm.. don't know really, was not inferring them for me